.github/workflows/cobalt_build.yml

```diff
@@ -0,0 +1,29 @@
+name: Build and Deploy Blog
```

Comment:
I noticed this work. Nice job! It appears though that the diagram is in error. Can you rectify?